elixir_server/README.md

```diff
@@ -0,0 +1,46 @@
+# ElixirServer
+
+**TODO: Add description**
+
+## Installation
+
+If [available in Hex](https://hex.pm/docs/publish), the package can be installed
+by adding `elixir_server` to your list of dependencies in `mix.exs`:
+
+```elixir
+def deps do
+  [
+    {:elixir_server, "~> 0.1.0"}
+  ]
+end
+```
+
+Documentation can be generated with [ExDoc](https://github.com/elixir-lang/ex_doc)
+and published on [HexDocs](https://hexdocs.pm). Once published, the docs can
+be found at <https://hexdocs.pm/elixir_server>.
+
+## Info
+
+Download tesseract:
+<https://digi.bib.uni-mannheim.de/tesseract/tesseract-ocr-w64-setup-v4.1.0.20190314.exe>
```

Comment:
Also works on tesseract 5.0
Maybe add link to https://tesseract-ocr.github.io/tessdoc/Installation.html

Reply:
API is fully compatible with 4.x.x we don't know if everything will be working with 5.0